### ISSUE TITLE
We can come aboard the codeship? 

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/backend#302]
Hey Mark, occationally @chealsye & I push builds that fail a codeship test. I'm assuming that if we're on the account driving these tests i'll tell us what's up & let us remedy (right now clicking the red x just says we're not authorized)? 

Can we be added to the account?

**Comment by markmacgillivray** [comment]
I could add you - if you both sign up to codeship and give me your account names I can add you to the project. But it wouldn't help - the clashes occur when there is a discrepancy between something on the server and something in the new code to be pushed, and that can't be fixed from codeship. Regarding the recent failures, I have just pushed a commit to fix that merge conflict.

**Comment by JosephMcArthur** [comment]
Got it, okay, then I'm not keen on being on it if it wont help :)

**Comment by JosephMcArthur** [comment]
What's the best way to resolve merge conflicts? Just ping you, or can we do something ourselves?

**Comment by markmacgillivray** [comment]
I get notifications about failures from codeship, so then I check. You
could ping me, but by the time you do I will probably have already seen an
email from codeship. Then it is just a case of me fixing as quick as
possible, which is usually pretty quick. This morning I think I was asleep
when the conflicts arose, and it was due to me changing the front page in
ways that we then decided not to keep, but I didn't push those changes
before going to sleep, sorry.
